app/controllers/days_controller.rb

```diff
@@ -7,16 +7,9 @@ def new
   end
 
   def create
-    current_day = Day.current_for(current_instructor.current_cohort)
     day = Day.new(day_params)
     day.cohort = current_instructor.current_cohort
     if day.save
-      current_day.unaccounted_for_students.each do |student|
```

Comment:
I realize this is throwing an error when creating a new account that needs to be fixed. The `first_day` concept makes sense. 

Maybe you moved this somewhere and I don't see it, but one thing I don't want to lose is the concept of attendance tracking through the app by adding new days. The idea being that every time you start a new day in the morning, all students who had not checked are now marked as absent for that day.